throughput/spam.go

```diff
@@ -0,0 +1,427 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+//nolint:gosec
+package throughput
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"os"
+	"os/signal"
+	"runtime"
+	"sync"
+	"sync/atomic"
+	"syscall"
+	"time"
+
+	"github.com/ava-labs/avalanchego/utils/set"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/ava-labs/hypersdk/api/jsonrpc"
+	"github.com/ava-labs/hypersdk/api/ws"
+	"github.com/ava-labs/hypersdk/auth"
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/codec"
+	"github.com/ava-labs/hypersdk/consts"
+	"github.com/ava-labs/hypersdk/fees"
+	"github.com/ava-labs/hypersdk/pubsub"
+	"github.com/ava-labs/hypersdk/utils"
+)
+
+const (
+	amountToTransfer               = 1
```

Comment:
Is there a reason for `amountToTransfer` replacing `feePerTx` in a few places?

Reply:
where do you see it replacing `feePerTx`? I only added it and it shouldn't be replacing it anywhere

Reply:
Ah I misread, is this just to account for the balance transfer that wasn't accounted for before here: https://github.com/ava-labs/hypersdk/blob/b8104aca01c2025dfb00f847d49e3a4d5d36648c/cli/spam.go#L372 ?

Reply:
yep!